helix-rest/src/test/java/org/apache/helix/rest/server/TestResourceAccessor.java

```diff
@@ -657,19 +660,9 @@ private void createDummyMapping(String clusterName, String resourceName,
     System.out.println("End test :" + TestHelper.getTestMethodName());
   }
 
-  private void stopClustersControllers() {
-    for (ClusterControllerManager cm : _clusterControllerManagers) {
-      if (cm != null && cm.isConnected()) {
-        cm.syncStop();
-      }
-    }
+  private void enableCluster(String clusterName, boolean enable) {
+    HelixAdmin helixAdmin = new ZKHelixAdmin(_gZkClient);
+    helixAdmin.enableCluster(clusterName, enable);
```

Comment:
Why do you need to create a HelixAdmin object every time this is called?

Reply:
Also, don't we already have a HelixAdmin in the parent abstract class created with _gZkClient already?

Reply:
+1. I believe `_gSetupTool.getClusterManagementTool()` gives you helix admin and this function `enableCluster()`. So `_gSetupTool.getClusterManagementTool().enableCluster()` is one line you need. This private method may not be needed.

Reply:
Wasn't able to find HelixAdmin in the parent class, but @pkuwm is correct. Thank you for the pointers and updated accordingly!